packages/zoe/src/zoeService/createZCFVat.js

```diff
@@ -0,0 +1,20 @@
+import { E } from '@agoric/eventual-send';
+
+import zcfContractBundle from '../../bundles/bundle-contractFacet';
+
+/**
+ * Attenuate the power of vatAdminSvc by restricting it such that only
+ * ZCF Vats can be created.
+ *
+ * @param {VatAdminSvc} vatAdminSvc
+ * @param {string=} zcfBundleName
+ * @returns {CreateZCFVat}
+ */
+export const setupCreateZCFVat = (vatAdminSvc, zcfBundleName) => {
+  /** @type {CreateZCFVat} */
+  const createZCFVat = () =>
+    zcfBundleName
```

Comment:
When a parameter is optional I like to see it in its declaration as well.
```suggestion
export const setupCreateZCFVat = (vatAdminSvc, zcfBundleName = undefined) => {
  /** @type {CreateZCFVat} */
  const createZCFVat = () =>
    zcfBundleName !== undefined
```
or
```js
    typeof zcfBundleName === 'string'
```
depending on your intent, and on the requirements and assumptions of `createVatByName`.

Reply:
Thanks for this. Done!